cmd/mount_unix.go

```diff
@@ -153,7 +154,21 @@ func mount_flags() []cli.Flag {
 
 func disableUpdatedb() {
 	path := "/etc/updatedb.conf"
-	data, err := os.ReadFile(path)
+	file, err := os.Open(path)
+	if err != nil {
+		return
+	}
+	defer file.Close()
+
+	// obtain exclusive and not block flock
+	if err := syscall.Flock(int(file.Fd()), syscall.LOCK_EX|syscall.LOCK_NB); err != nil {
```

Comment:
Is it possible that Flock is not supported?

Reply:
some fs maybe not support flock, It's very rare.  I added condition if err is not syscall.EAGAIN (resource temporarily unavailable), it not use flock